### PR TITLE
chore: intro `install-dist` module in root

### PR DESCRIPTION
**Pull request by VGalaxies**
- intro `install-dist` module in root
- move release-docs and scripts from `hugegraph-server/hugegraph-dist` to `install-dist`
- change package artifact name to `apache-hugegraph-incubating-pd/server-xxx`

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -93,6 +93,7 @@
         <module>hugegraph-server</module>
         <module>hugegraph-pd</module>
         <module>hugegraph-store</module>
+        <module>dist</module>
```
[the remark this answers is not in the document]
Comment (VGalaxies) [reply pom.xml:96]: The module name "hugegraph-dist" is currently taken... Consider renaming after "hugegraph-dist" is renamed to "server-dist".

**Comment by VGalaxies** [comment]
Found LICENSE in jar package:
- [ ] commons-text-1.9
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] hg-pd-core-1.3.0
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jackson-datatype-jsr310-2.12.6
    - found jar package
    - not found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jakarta.annotation-api-1.3.5
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jakarta.websocket-api-1.1.2
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] javax-websocket-client-impl-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] javax-websocket-server-impl-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-annotations-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-client-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-continuation-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-http-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-io-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-plus-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-security-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-server-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-servlet-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-servlets-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-util-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-util-ajax-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-webapp-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jetty-xml-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] jna-5.5.0
    - found jar package
    - not found NOTICE in jar package
    - found LICENSE in jar package
- [ ] log4j-api-2.17.2
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] log4j-core-2.17.2
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] log4j-jul-2.17.2
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] log4j-slf4j-impl-2.17.0
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] micrometer-core-1.7.12
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] micrometer-registry-prometheus-1.7.12
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-aop-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-beans-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-boot-2.5.14
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-boot-actuator-2.5.14
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-boot-actuator-autoconfigure-2.5.14
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-boot-autoconfigure-2.5.0
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-context-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-core-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-expression-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-jcl-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-web-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] spring-webmvc-5.3.20
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] tomcat-embed-el-9.0.63
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] websocket-api-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] websocket-client-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] websocket-common-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] websocket-server-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package
- [ ] websocket-servlet-9.4.46.v20220331
    - found jar package
    - found NOTICE in jar package
    - found LICENSE in jar package

Other cases:
- [ ] android-json-0.0.20131108.vaadin1
    - not found jar package
- [ ] apiguardian-api-1.1.0
    - not found jar package
- [ ] asm-analysis-9.2
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] asm-commons-9.2
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] asm-tree-9.2
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] assertj-core-3.19.0
    - not found jar package
- [ ] byte-buddy-1.10.20
    - not found jar package
- [ ] byte-buddy-agent-1.10.20
    - not found jar package
- [ ] commons-io-2.8.0
    - not found jar package
- [ ] disruptor-3.4.1
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] error_prone_annotations-2.4.0
    - not found jar package
- [ ] fastjson-1.2.83
    - not found jar package
- [ ] grpc-services-1.39.0
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] grpc-spring-boot-starter-4.5.5
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] grpc-stub-1.39.0
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] gson-2.8.6
    - not found jar package
- [ ] guava-30.1-android
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] HdrHistogram-2.1.12
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] hg-pd-dist-1.3.0
    - not found jar package
- [ ] hg-pd-service-1.3.0
    - not found jar package
- [ ] jackson-annotations-2.13.0
    - not found jar package
- [ ] jackson-core-2.13.0
    - not found jar package
- [ ] jackson-databind-2.13.0
    - not found jar package
- [ ] jackson-datatype-jdk8-2.12.6
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] jackson-module-parameter-names-2.12.6
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] jakarta.servlet-api-4.0.4
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] jakarta.xml.bind-api-2.3.3
    - not found jar package
- [ ] javassist-3.24.0-GA
    - not found jar package
- [ ] javassist-3.28.0-GA
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] jraft-core-1.3.13
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] jraft-core-1.3.9
    - not found jar package
- [ ] jsonassert-1.5.0
    - not found jar package
- [ ] json-path-2.5.0
    - not found jar package
- [ ] jsr305-3.0.2
    - not found jar package
- [ ] jul-to-slf4j-1.7.36
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] junit-4.13.2
    - not found jar package
- [ ] junit-jupiter-5.7.2
    - not found jar package
- [ ] junit-jupiter-api-5.7.2
    - not found jar package
- [ ] junit-jupiter-engine-5.7.2
    - not found jar package
- [ ] junit-jupiter-params-5.7.2
    - not found jar package
- [ ] junit-platform-commons-1.7.2
    - not found jar package
- [ ] junit-platform-engine-1.7.2
    - not found jar package
- [ ] LatencyUtils-2.0.3
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] log4j-api-2.15.0
    - not found jar package
- [ ] log4j-api-2.17.0
    - not found jar package
- [ ] log4j-api-2.18.0
    - not found jar package
- [ ] log4j-core-2.15.0
    - not found jar package
- [ ] log4j-core-2.17.0
    - not found jar package
- [ ] log4j-core-2.18.0
    - not found jar package
- [ ] log4j-slf4j-impl-2.15.0
    - not found jar package
- [ ] lombok-1.18.24
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] mockito-core-3.9.0
    - not found jar package
- [ ] mockito-junit-jupiter-3.9.0
    - not found jar package
- [ ] netty-all-4.1.42.Final
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] netty-tcnative-boringssl-static-2.0.25.Final
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] objenesis-3.2
    - not found jar package
- [ ] opentest4j-1.2.0
    - not found jar package
- [ ] perfmark-api-0.23.0
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] powermock-api-mockito2-2.0.0-RC.3
    - not found jar package
- [ ] powermock-api-support-2.0.0-RC.3
    - not found jar package
- [ ] powermock-classloading-base-2.0.0-RC.3
    - not found jar package
- [ ] powermock-classloading-xstream-2.0.0-RC.3
    - not found jar package
- [ ] powermock-core-2.0.0-RC.3
    - not found jar package
- [ ] powermock-module-junit4-2.0.0-RC.3
    - not found jar package
- [ ] powermock-module-junit4-common-2.0.0-RC.3
    - not found jar package
- [ ] powermock-module-junit4-rule-2.0.0-RC.3
    - not found jar package
- [ ] powermock-reflect-2.0.0-RC.3
    - not found jar package
- [ ] protobuf-java-3.17.2
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] protobuf-java-3.5.1
    - not found jar package
- [ ] rocksdbjni-6.29.5
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] rocksdbjni-7.7.3
    - not found jar package
- [ ] simpleclient-0.10.0
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] simpleclient_common-0.10.0
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] slf4j-api-1.7.21
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] snakeyaml-1.28
    - found jar package
    - not found NOTICE in jar package
    - not found LICENSE in jar package
- [ ] spring-boot-autoconfigure-2.5.14
    - not found jar package
- [ ] spring-boot-starter-2.5.0
    - not found jar package
- [ ] spring-boot-starter-2.5.14
    - not found jar package
- [ ] spring-boot-starter-actuator-2.5.14
    - not found jar package
- [ ] spring-boot-starter-jetty-2.5.14
    - not found jar package
- [ ] spring-boot-starter-json-2.5.14
    - not found jar package
- [ ] spring-boot-starter-log4j2-2.5.14
    - not found jar package
- [ ] spring-boot-starter-test-2.5.14
    - not found jar package
- [ ] spring-boot-starter-web-2.5.14
    - not found jar package
- [ ] spring-boot-test-2.5.14
    - not found jar package
- [ ] spring-boot-test-autoconfigure-2.5.14
    - not found jar package
- [ ] spring-context-support-5.3.20
    - not found jar package
- [ ] xmlpull-1.1.3.1
    - not found jar package
- [ ] xmlunit-core-2.8.4
    - not found jar package
- [ ] xpp3_min-1.1.4c
    - not found jar package
- [ ] xstream-1.4.10
    - not found jar package

File: .gitignore [hunk 2]
```diff
@@ -31,7 +31,7 @@ gen-java
 /nbbuild/
 /nbdist/
 /.nb-gradle/
-dist/
```
Comment (imbajin) [inline .gitignore:34]: WARNING

File: hugegraph-pd/hg-pd-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -52,13 +52,13 @@ pd:
...
   # The initial store list is automatically activated in the store
-  initial-store-list: 127.0.0.1:8502
+  initial-store-list: 127.0.0.1:8500
 
 raft:
   # The address of the local raft service
   address: 127.0.0.1:8610
   # The service address of the PD cluster
-  peers-list: 127.0.0.1:8610,127.0.0.1:8611,127.0.0.1:8612
+  peers-list: 127.0.0.1:8610
```
Comment (imbajin) [inline hugegraph-pd/hg-pd-service/src/main/resources/application.yml:61]: Any potential impact?

**Review by imbajin — APPROVED** [review]

File: hugegraph-pd/hg-pd-service/src/main/resources/application.yml (hunk 3 above)
Comment (VGalaxies) [reply hugegraph-pd/hg-pd-service/src/main/resources/application.yml:61]: Currently, no side effects have been found. The modification is intended to use `org.apache.hugegraph.pd.boot.HugePDServer` to quickly start a single node PD.

**Review by liuxiaocs7 — APPROVED** [review]

**Pull request merged**